File: judge_templates/php/judge.php
#!/usr/bin/php -c/usr/etc
<?php

array_shift($argv);
list($referenceFile, $resultFile) = $argv;

$errors = [];

/*
 * Check the results....
 */


// Print the judgement.
if ($errors) {
	echo "0.0\n";
	echo join("\n", $errors), "\n";
	exit(1);
}
else {
	echo "1.0\n";
	exit(0);
}

File: judge_templates/python/judge.py
#!/usr/bin/env python3

import sys

[referenceFile, resultFile] = sys.argv[1:] 
error = ""

#
# Compare the results....
# 

# Print the output.
if (error):
    print("0.0")
    print(error)
    sys.exit(1)
else:
    print("1.0")
    sys.exit(0)
